email-template.txt

```diff
@@ -0,0 +1,27 @@
+To: {name} <{email}>
+From: DataLad Team <team@datalad.org>
+Subject: Invitation to co-author a DataLad paper for JOSS
+
+Dear {name},
+
+Thank you for your previous contribution to DataLad (https://github.com/datalad/datalad), by code, issues, or feedback.
+We are working on a manuscript to be submitted to JOSS (https://joss.theoj.org) to describe DataLad, and would like to
+acknowledge your contribution(s).  We are inviting you to co-author the paper, or, alternatively, give us permission to
+thank you in the Acknowledgements section of the paper.
+
+If you would like to co-author the paper, please submit a Pull Request with your details (name, ORCID, affiliation)
```

Comment:
@mih if you could suggest a diplomatic wording about responsibilities of the co-author, would be appreciated. Otherwise I will try tomorrow

Reply:
Thank you!